models/commentsModel.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection")
+
+exports.getAllCommentsForArticleModel = async (article_id) => {
+  const comments = await db.query(`
+    SELECT *
+    FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+  `, [article_id])
+
+  if (comments.rows.length < 1) return Promise.reject({errCode: 404, errMsg: "Article ID not found"})

```

Comment:
This isn't a good method to check if the article ID isn't found.
If there is a valid articleID but no comments we should still send those back.